src/identities/CommandClaim.ts

```diff
@@ -70,25 +66,30 @@ class CommandClaim extends CommandPolykey {
           nodePath: options.nodePath,
           logger: this.logger.getChild(PolykeyClient.name),
         });
-        const claimMessage = await binUtils.retryAuthentication(
-          (auth) =>
-            pkClient.rpcClientClient.methods.identitiesClaim({
-              metadata: auth,
-              providerId: providerId,
-              identityId: identityId,
-            }),
-          auth,
-        );
-        const output = [`Claim Id: ${claimMessage.claimId}`];
-        if (claimMessage.url) {
-          output.push(`Url: ${claimMessage.url}`);
+        const [type, id] = gestaltId;
+        switch (type) {
+          case 'identity': {
+            const claimMessage = await binUtils.retryAuthentication(
+              (auth) =>
+                pkClient.rpcClientClient.methods.identitiesClaim({
+                  metadata: auth,
+                  providerId: id[0],
+                  identityId: id[1],
+                }),
+              auth,
+            );
+            const output = [`Claim Id: ${claimMessage.claimId}`];
+            if (claimMessage.url) {
+              output.push(`Url: ${claimMessage.url}`);
+            }
+            process.stdout.write(
+              binUtils.outputFormatter({
+                type: options.format === 'json' ? 'json' : 'list',
+                data: output,
+              }),
+            );
+          }
```

Comment:
This is really bad, I understand that you want to enforce the type for an `IdentityId`. BUT a switch statement with only 1 case is really bad way to do this.

1. We shouldn't be using the `GestaltId` here, we should only be working with an `IdentityId` here.
2. You cant use a single case for a switch statement like this. It's basically saying only handle the `identity` type and do nothing in all other cases. If someone provides a `NodeId` this block will do nothing, If for some reason anything else was provided (it can happen, thats a whole rant about typescript to go into) it will do nothing. No explination, just nothing happens. That is not a fun bug to track down.
3. If you ever use a switch statement. ALWAYS include a default case that calls `utils.never()` even if all cases are covered. This is a fail safe.


Ultimately you shouldn't *need* a switch statement here, you shouldn't be using the `parseGestaltId`.

Also, did this not feel wrong at all when writing it? This is a pretty bad code smell.